app/views/claims/claim_school.html.erb

```diff
@@ -3,57 +3,62 @@
 
     <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
 
-    <h1 class="govuk-heading-xl">
-      <%= t("tslr.questions.claim_school") %>
-    </h1>
-
     <% if @schools.blank? %>
```

Comment:
At this point, these might as well be two separate views, since the two cases share next to nothing.

Reply:
Does that have any impact on the autocomplete?

Reply:
Well, it involves changing the controller, so yes, some.

Reply:
I think I will leave this for now, we kept it all in the same view for simplicity and we can always seperate it later.